src/nav/components.yml

```diff
@@ -22,3 +22,7 @@
           title: Multiselect
     - title: Tabs
       id: /components/tabs
+    - title: Text inputs
+      items:
+        - id: /components/text-inputs
+          title: Text inputs
```

Comment:
In conjunction with @austin94's "Input" comment, the IA feels off here. What is meant to be contained under this "Text inputs" parent?

Reply:
Oops, IA is off and I can make the appropriate adjustments. The concerns around "Text inputs" were brought up in the Google doc yesterday.

Reply:
You can follow this IA structure for the placement of this: https://5ee276003c482c6da838de83--zendeskgarden.netlify.app/components

Reply:
Updated page title from "Text inputs" to "Input" and IA.